pyproject.toml

```diff
@@ -15,22 +15,27 @@ force_single_line = true
 line_length = 80
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.11.2"
 astral = "^2.2"
 google-api-python-client = "^1.12.8"
 google-auth-httplib2 = "^0.0.4"
 google-auth-oauthlib = "^0.4.2"
 pydantic = "^1.7.3"
 click = "^8.0.1"
+rsa = "*"
 
 [tool.poetry.dev-dependencies]
-pytest = "^5.2"
-black = "^20.8b1"
+pytest = "^7.2.1"
+black = "^22.3.0"
 isort = "^5.8.0"
-mypy = "^0.812"
+mypy = "^1.0.0"
 ipykernel = "^5.5.0"
 jupyter = "^1.0.0"
 pylint = "^2.7.4"
+types-pytz = "^2022.7.1.0"
+types-setuptools = "^67.3.0.1"
+wrapt = "^1.14.1"
```

Comment:
Why do we need `wrapt` and `dill`?

Reply:
We didn't until I updated pytest. When I brought pytest to a more contemporary version it demanded these two dependencies.